crates/ink/tests/ui/event/fail/conficting_attributes_inline.rs

```diff
@@ -0,0 +1,7 @@
+#[ink::event]
+#[ink(anonymous, signature_topic = "1111111111111111111111111111111111111111111111111111111111111111")]
```

Comment:
This hits a different error `Invalid `#[ink(anonymous)]` attribute: multiple arguments not allowed`, and not the clause you added which has a more useful error message. Maybe separate `ink` attributes?

Reply:
Well, technically, we don't allow to have multiple arguments inside `#[ink()]`. So this is a valid error. And what clause are you talking about specifically?